src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -422,6 +422,7 @@ void ScatterUpdate::scatterNDUpdate(uint8_t *indices, uint8_t *update, uint8_t *
             size_t idxValue = getIndicesValue(indices, indicesOffset + i);
             dstOffset += idxValue * srcBlockND[i + 1];
         }
+        dstOffset--;
```

Comment:
shall we out here `assert(dstOffet)` at least?
otherwise comment why we do need extract 1 - would be appreciated